src/server/routes/entity/creator.js

```diff
@@ -95,17 +96,13 @@ router.get(
 			'Creator'
 		);
 
-		const props = generateProps(req, res, {
-			entityType: 'creator',
-			genderOptions: res.locals.genders,
-			heading: 'Create Creator',
-			identifierTypes: filteredIdentifierTypes,
-			initialState: {},
-			languageOptions: res.locals.languages,
-			requiresJS: true,
-			subheading: 'Add a new Creator to BookBrainz',
-			submissionUrl: '/creator/create/handler'
-		});
+		const props = generateEntityProps(
+			'creator', 'create', req, res, {
+				genderOptions: res.locals.genders,
+				identifierTypes: filteredIdentifierTypes,
```

Comment:
Could we do identifier type filtering in the generateEntityProps function too? Seems like we have all the info there we need.

Reply:
Yeah, you're right.